app/javascript/app/modules/personal-key-formatter.js

```diff
@@ -1,45 +0,0 @@
-import { DelimitedTextFormatter } from 'field-kit';
-
-/**
- * @const
- * @private
- */
-const DIGITS_PATTERN = /^[a-zA-Z0-9]*$/;
-
-/**
- * @extends DelimitedTextFormatter
- */
-class PersonalKeyFormatter extends DelimitedTextFormatter {
```

Comment:
Do we have any examples of which specific field's backspace keys weren't working? I think some of the formatters like this for the personal key are nice touches, automatically adding the `-`'s

Reply:
https://cm-jira.usa.gov/browse/LG-2532 The SMS OTP field, for example.

Check out @solipet's comments on the JIRA ticket for how we arrived to this solution (I'm picking up where he left off). I wasn't part of this particular convo but I *think* he and @jmhooper decided it was favorable to just get rid of the formatters for now

Reply:
Ok thanks for the link, that's super useful thanks! Definitely agree then